action.yml

```
@@ -63,17 +63,25 @@ runs:
       with:
         base_ref_arg: ${{ github.event.inputs.base_ref }}
         head_ref_arg: ${{ github.event.inputs.head_ref }}
+        payload_arg: ${{ github.event.inputs.client_payload }}
         url_arg: ${{ fromJSON(fromJSON(github.event.inputs.client_payload)).headHttpUrl || fromJSON(fromJSON(github.event.inputs.client_payload)).repoUrl }}
         script: |
-          function escapeSingleQuoteChars(input) {
+          function escapeSingleQuoteInString(input) {
```

IMO - maybe change the function name to something more generic since it's a public repo
```suggestion
          function convertToSafeString(input) {
```